synfig-studio/src/gui/duckmatic.h

```diff
@@ -114,6 +114,13 @@ class BezierDrag_Default : public BezierDrag_Base
 	void bezier_drag(Duckmatic* duckmatic, const synfig::Vector& vector);
 };
 
+struct Guide
+	{
+		synfig::Point point;
+		synfig::Angle::rad angle;
+		bool isVertical;
```

Comment:
`isVertical` is never set. Why do you use it?
Even if it is needed, it should be a method: it's just a matter of check the angle.

Reply:
It is only set on ruler creation. It is needed as rulers generated from the vertical ruler have their center of rotations at the middle of the screens height, and the horizontal ruler's have their center of rotations around half the screen width. This is to make the rulers reach more places using just the gui/without the dialog.